core/src/main/java/com/googlecode/psiprobe/beans/stats/providers/MultipleSeriesProvider.java

```diff
@@ -36,7 +36,7 @@
 public class MultipleSeriesProvider extends AbstractSeriesProvider {
   
   /** The stat name prefix. */
-  private String statNamePrefix;
```

Comment:
Why remove this modifier? I tend to make everything private unless there's a compelling reason not to.